### PR TITLE
gh: auto lock closed threads

### PR DESCRIPTION
**Pull request by abn**

### CODE REVIEW

File: .github/workflows/lock-threads.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```
Comment (Secrus) [inline .github/workflows/lock-threads.yml:5]: Please add a description for this setting (cron settings are not that easy to remember).

File: .github/workflows/lock-threads.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v5
+        with:
+          issue-inactive-days: 30
+          issue-comment: >
+            This issue has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
+          pr-comment: >
```
Comment (Secrus) [inline .github/workflows/lock-threads.yml:27]: I wouldn't set this for PRs right now, some old PRs are good just abandoned and need some TLC)

**Review by Secrus — CHANGES_REQUESTED** [review]

File: .github/workflows/lock-threads.yml (hunk 2 above)
Comment (neersighted) [reply .github/workflows/lock-threads.yml:27]: I think this is fine -- this is not an auto-close, but an auto-lock. If there is a good old PR that is closed and unmerged, someone should carry the changes as a new PR anyway.

**Review by neersighted — APPROVED** [review]

File: .github/workflows/lock-threads.yml (hunk 1 above)
Comment (abn) [reply .github/workflows/lock-threads.yml:5]: [crontab.guru](https://crontab.guru/#0_*_*_*_* ) is handy. :)

> “At minute 0.”

Comment (abn) [inline .github/workflows/lock-threads.yml:5]:
```suggestion
    # at minute 0
    - cron: '0 * * * *'
```

**Review by neersighted — APPROVED** [review]

**Pull request merged**